src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -636,13 +636,46 @@ static void TransformationUpToCPUSpecificOpSet(std::shared_ptr<ngraph::Function>
 
     // Execute before snippets. Otherwise FQ will be converted to Subgraph
     postLPTPassManager.register_pass<ConvertFqRnnToQuantizedRnn>();
+
+    // Float MHA is supported by snippets now
+    auto postLPTPassConfig = postLPTPassManager.get_pass_config();
+    postLPTPassConfig->disable<MHAFloatFusion>();
+    postLPTPassConfig->disable<MHAFloatFusion2>();
```

Comment:
Since int8 MHA tokenization is not supported yet, shall we disable only in fp32 cases?

Reply:
You're right, it's fine as it is, my bad